pkgs/applications/editors/neovim/default.nix

```
@@ -113,6 +113,11 @@ in
       substituteInPlace src/nvim/CMakeLists.txt --replace "    util" ""
     '';
 
+    postFixup = ''
+      patchelf --add-needed ${gcc.cc.lib}/lib/libstdc++.so.6 $out/bin/nvim
```

```suggestion
      patchelf --add-needed ${stdenv.cc.cc.lib}/lib/libstdc++.so.6 $out/bin/nvim
```

might work for mac?

could also mark it as `lib.optionalString stdenv.isLinux ''` for now as well